src/screens/LoginScreen.tsx

```diff
@@ -0,0 +1,51 @@
+import { useState } from 'react';
+import { supabase } from '../helpers/supabaseClient';
+
+export default function LoginScreen() {
+	const [loading, setLoading] = useState(false);
+	const [email, setEmail] = useState('');
+	const [password, setPassword] = useState('');
+
+	async function signInWithEmail(e: any) {
+		e.preventDefault();
+		const { data, error } = await supabase.auth.signInWithPassword({
+			email: email,
+			password: password,
+		});
+	}
+
+	return (
+		<div className="row flex-center flex">
+			<div className="col-6 form-widget" aria-live="polite">
+				<h1 className="header">Supabase + React</h1>
+				<p className="description">Login</p>
+				{loading ? (
+					'Sending magic link...'
+				) : (
+					<form onSubmit={signInWithEmail}>
+						<label htmlFor="email">Email</label>
+						<input
+							id="email"
+							className="inputField"
+							type="email"
+							placeholder="Your email"
+							value={email}
+							onChange={(e) => setEmail(e.target.value)}
+						/>
+						<input
+							id=""
+							className="inputField"
+							type="password"
+							placeholder="Your pasword"
+							value={password}
+							onChange={(e) => setPassword(e.target.value)}
+						/>
+						<button className="button block" aria-live="polite">
```

Comment:
This works, but why does it not have to be `type="submit"` ?

Reply:
@Thenlie Default button type is "submit".

submit: The button submits the form data to the server. This is the default if the attribute is not specified for buttons associated with a <form>, or if the attribute is an empty or invalid value.
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button